cfgmgr/intfmgr.cpp

```diff
@@ -731,8 +743,8 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
         std::vector<FieldValueTuple> fvVector;
         FieldValueTuple f("family", ip_prefix.isV4() ? IPV4_NAME : IPV6_NAME);
 
-        // Don't send link local config to AppDB and Orchagent
-        if (ip_prefix.getIp().getAddrScope() != IpAddress::AddrScope::LINK_SCOPE)
+        // Don't send ipv4 link local config to AppDB and Orchagent
+        if ((ip_prefix.isV4() == false) or (ip_prefix.getIp().getAddrScope() != IpAddress::AddrScope::LINK_SCOPE))
```

Comment:
Minor comment - Please use "||" instead for uniformity. Same as below.

Reply:
Done.